upstairs/src/volume.rs

```diff
@@ -2694,15 +2615,17 @@ mod test {
 
         assert_eq!(volume.total_size().await?, block_size as u64 * 10);
 
-        // Verify parent contents in one read
-        let buffer = Buffer::new(block_size * 10);
-        volume
-            .read(Block::new(0, block_size.trailing_zeros()), buffer.clone())
+        // Verify volume contents in one read
+        let buffer = Buffer::new(block_size * 1);
```

Comment:
This seems like you're testing a different thing now?

Reply:
excellent catch, reverted in 31048fa